src/NodeCollector/NodeAnalyzer/ArrayCallableMethodMatcher.php

```diff
@@ -59,7 +62,7 @@ public function match(Array_ $array, Scope $scope): null | ArrayCallableDynamicM
         // $this, self, static, FQN
         $firstItemValue = $items[0]->value;
 
-        $callerType = $this->resolveCallerType($firstItemValue, $scope);
+        $callerType = $this->resolveCallerType($firstItemValue, $scope, $classMethodName);
```

Comment:
I am on mobile, looks like method name should can be pulled from ` $items[1]->value->value` directly if `$items[1]->value` is a `String_` node

Reply:
Thank you for your reply. I tried but PHPUnit failed. (https://github.com/rectorphp/rector-src/pull/5770#discussion_r1536821064)

Reply:
I see, it seems due to current class method vs in 2nd arg is different 👍